Archive the repository

Kijewski opened the pull request:
This is the old repository of the [askama](https://crates.io/crates/askama) project.

Please go to **<https://github.com/askama-rs/askama>** to see the current state of the project.

GuillaumeGomez commented:
I'm not sure we should remove everything. Isn't updating the README content enough?

Kijewski committed:
Archive the repository

This is the old repository of the [askama] project.

Please go to **<https://github.com/askama-rs/askama>** to see the
current state of the project.

[askama]: https://crates.io/crates/askama

GuillaumeGomez commented:
Thanks!

Kijewski commented:
Yes, makes sense to keep the tree. I was just thinking how to promote the new URL the best. But most people will likely scroll down to the README first anyway, even if there is a lot of code in the tree.
